dbt/include/cratedb/macros/materializations/seed.sql

```diff
@@ -0,0 +1,13 @@
+{# CrateDB: Use `DELETE FROM` instead of `TRUNCATE` #}
+{% macro cratedb__reset_csv_table(model, full_refresh, old_relation, agate_table) %}
+    {% set sql = "" %}
+    {% if full_refresh %}
+        {{ adapter.drop_relation(old_relation) }}
+        {% set sql = create_csv_table(model, agate_table) %}
+    {% else %}
+        {{ adapter.truncate_relation(old_relation) }}
+        {% set sql = "delete from " ~ old_relation.render() %}
+    {% endif %}
+
+    {{ return(sql) }}
+{% endmacro %}
```

Comment:
I've derived this from @hlcianfagna's contributions at [Using dbt with CrateDB](https://community.cratedb.com/t/using-dbt-with-cratedb/1566). ~~However, it has not been activated yet, and I don't know yet what it is about / which code paths, subsystems, and test cases it could unlock.~~

- https://github.com/crate/cratedb-examples/pull/754#discussion_r1862067486

This has been activated now, accompanied by a corresponding software test. See next comment.